Dockerfile

```
@@ -16,18 +15,52 @@ RUN apt-get -y update
 RUN apt-get -y install git wget xz-utils build-essential \
                        curl dejagnu subversion bison flex bc libcap-dev \
                        cmake libncurses5-dev libboost-all-dev \
-                       default-jdk sbt
+                       default-jdk sbt --no-install-recommends
```

```suggestion
                       sbt --no-install-recommends
```

Remove installing `default-jdk` here as it will be purged later.